### PR TITLE
add more connection stats logging

### PR DESCRIPTION
**Pull request by alexkorotkikh**
add more connection stats logging

### CODE REVIEW

**Comment by rajatparida86** [comment]
👍

File: src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java [hunk 1]
```diff
@@ -107,13 +108,14 @@ private void logConnectionStats(final Session session) {
...
-			for (Host host : state.getConnectedHosts()) {
+			final Collection<Host> connectedHosts = state.getConnectedHosts();
+			for (Host host : connectedHosts) {
 				final HostDistance distance = configuration.getPolicies().getLoadBalancingPolicy().distance(host);
                 final int connections = state.getOpenConnections(host);
                 final int inFlightQueries = state.getInFlightQueries(host);
 				final int maxRequestsPerConnection = poolingOptions.getMaxRequestsPerConnection(distance);
-                logger.debug("connection_stats: host={} connections={}, current_load={}, max_load={}",
-                        host, connections, inFlightQueries, connections * maxRequestsPerConnection);
+                logger.debug("connection_stats: hosts_count={} host={} distance = {} connections={}, current_load={}, max_load={}",
```
Comment (vetinari) [inline src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java]: please drop the spaces around the `=` after `distance`

Comment (alexkorotkikh) [reply src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java]: sure, it was done accidentally. Thanks

**Comment by alexkorotkikh** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Pull request merged**